examples/neq-switching/run_nonequilibrium.py

```diff
@@ -9,13 +9,13 @@
 from perses.dispersed.feptasks import compute_reduced_potential
 import numpy as np
 
-def traj_frame_to_sampler_state(traj: md.Trajectory, frame_number: int):
+def traj_frame_to_sampler_state(traj: md.Trajectory, frame_number: int,box_vectors):
     xyz = traj.xyz[frame_number, :, :]
     box_vectors = traj.openmm_boxes(frame_number)
-    sampler_state = states.SamplerState(unit.Quantity(xyz, unit=unit.nanometers), box_vectors=box_vectors)
+    sampler_state = states.SamplerState(unit.Quantity(xyz, unit=unit.nanometers))
     return sampler_state
 
-def run_rj_proposals(top_prop, configuration_traj, use_sterics, ncmc_nsteps, n_replicates, temperature=300.0*unit.kelvin):
+def run_rj_proposals(top_prop, configuration_traj, use_sterics, ncmc_nsteps, n_replicates, box_vectors, temperature=300.0*unit.kelvin):
```

Comment:
Don't need `box_vectors` as a separate argument---instead, will want to take them from the mdtraj Trajectory

Reply:
Will also need to fix calls to this because positional args are changing; may want to use kwargs there